ruby_on_rails/app_initialisation.md

```diff
@@ -39,12 +44,6 @@ If you do not need a DB you may rethink the fact that you may not need Rails at
 The following scripts are standardized tools for more convenience at Renuo.
 They are always idempotent (runnable multiple times).
 
-* Add a `bin/run` file. It will be used to start our project.
```

Comment:
why is this gone? 😢

Reply:
😭

Reply:
TBH: I don't mind running bin/dev ^^ alias br = 'bin/run' => alias br = 'bin/dev'

Reply:
What I think is worse, is that we now have projects with `bin/run` and `bin/dev` which sometimes do different stuff (bin/run just does rails s and bin/dev starts foreman)